components/psi/build.gradle.kts

```diff
@@ -1,15 +1,25 @@
 plugins {
     alias(libs.plugins.kotlin.jvm)
+    alias(libs.plugins.jetbrains.intellij.module)
 }
 
+repositories {
+    mavenCentral()
+
+    intellijPlatform {
+        defaultRepositories()
+    }
+}
 dependencies {
     implementation(projects.components.extensions)
 
-    // https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#kotlin-standard-library
-    compileOnly(libs.kotlin.stdlib)
-
-    implementation(libs.kotlin.embeddable)
-
     testImplementation(libs.bundles.test)
     testRuntimeOnly(libs.junit.launcher)
+
+    intellijPlatform {
+        intellijIdeaCommunity("2024.1")
+        instrumentationTools()
+
+        bundledPlugin("org.jetbrains.kotlin")
```

Comment:
Looks like these dependencies can't be accessed from test sourceset.

Reply:
the problem for now is how to use `com.intellij.openapi.project.Project` in tests 🙂 
will investigate

Reply:
@Goooler could you please check when you have time?

Reply:
A small tweak in f97742986283d03ab693987ef8bb4d5ade6bca0e.